internal/referenceapp/golang/client-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDZGBoWIoc9Xnvk
```

Comment:
Is checking in private keys okay? I believe this will be flagged by some security tool right?

Reply:
That is a great point. let me remove these and add to the One note for Mtls, will add the link to that in a bit.